11-spatial-cv.Rmd

```diff
@@ -259,8 +259,8 @@ For this, however, you will have to resort most likely to Bayesian modeling appr
 Spatial prediction maps are one very important outcome of a model.
 Even more important is how good the underlying model is at making them since a prediction map is useless if the model's predictive performance is bad.
 The most popular measure to assess the predictive performance of a binomial model is the Area Under the Receiver Operator Characteristic Curve (AUROC).
-This is a value between 0.5 and 1.0 with 0.5 indicating no and 1.0 indicating a perfect discrimination of the two classes. 
-Thus, the higher the AUROC the better is our model at making predictions.
+This is a value between 0.5 and 1.0 with 0.5 indicating por model fit and 1.0 indicating a perfect discrimination of the two classes. 
+Thus, the higher the AUROC the better our model is at making predictions.
```

Comment:
Definite improvement but I think there should be another comma, like this:
`Thus, the higher the AUROC, the better our model is at making predictions.`